pages/public_cloud/public_cloud_network_services/tutorial-stormshield_network_security_vrack/guide.fr-fr.md

```diff
@@ -0,0 +1,560 @@
+>
+
+## Prérequis
+
+- Un [projet Public Cloud](/pages/public_cloud/compute/create_a_public_cloud_project) dans votre compte OVHcloud.
+- Être connecté à l'[espace client OVHcloud](/links/manager).
+- Un [utilisateur OpenStack](/pages/public_cloud/compute/create_and_delete_a_user) (facultatif).
+- Des connaissances de base en réseau.
+- Un compte Stormshield créé via le [site Stormshield](https://www.stormshield.com/fr/){.external}.
+- S'assurer que le vRack est activé et configuré pour permettre une communication sécurisée entre les composants de l'infrastructure.
+- Une adresse [Additional IP](/links/network/additional-ip) pour permettre le failover et la configuration de la haute disponibilité.
+- Une licence Stormshield Network Security BYOL (**B**ring **Y**our **O**wn **L**icence), obtenue auprès de [partenaires ou revendeurs tiers](https://www.stormshield.com/partner/partner-finder/){.external}, que vous devrez fournir lors de l'installation et de la configuration.
+
+## En pratique
+
+En plus de l'installation et de la configuration de Stormshield Network Security, ce tutoriel présente différents cas d'usage en fonction de vos besoins :
+
+- [Installer et configurer Stormshield Network Security sur votre environnement Public Cloud](#step1)
+- [Cas d'usage 1 : configurer Stormshield Network Security pour une utilisation en tant que passerelle](#step2)
+- [Cas d'usage 2 : configurer un NAT (**N**etwork **A**ddress **T**ranslation) pour accéder à un service HTTP privé de l'extérieur](#step3)
+- [Cas d'usage 3 : tunnel IPsec (site à site)](#step4)
+- [Cas d'usage 4 : VPN SSL/TLS (client à site)](#step5)
+
+### Installer et configurer Stormshield Network Security sur votre environnement Public Cloud <a name="step1"></a>
+
+> [!primary]
+> Dans ce tutoriel, l'installation et la configuration de Stormshield Network Security (SNS) s'effectuent principalement via la ligne de commande. Ouvrez un terminal pour exécuter les instructions.
+>
+> Veuillez noter que toutes les rubriques relatives à la « Haute disponibilité » ou à « Stormshield-2 » sont facultatives, de même que l'utilisation du réseau vRack avec Additional IP. Ils sont inclus pour montrer comment mettre en place le système avec deux instances en mode actif/passif pour une haute disponibilité. Dans une version minimale, il peut également fonctionner avec une seule instance si cela suffit à vos besoins.
+
+> [!primary]
+> Dans ce scénario, nous utiliserons deux machines virtuelles configurées pour l'appliance de sécurité afin d'atteindre la haute disponibilité (**H**igh **A**vailability ou HA) et une machine virtuelle supplémentaire pour l'administration. Cette configuration assure une protection du failover et une disponibilité continue du service. Pour plus d'exemples et de conseils détaillés sur les options d'évolutivité, veuillez consulter la [documentation de Stormshield](https://documentation.stormshield.eu/HOME/Content/Website_Topics/Root-HomePage-FR.htm){.external}.
```

Comment:
Original EN version: "This setup ensures failover protection"
Your original FR translation @fabienOVH: "Cette configuration assure une protection contre le basculement"
My version: Cette configuration assure une protection du failover

I'm not convinced my version is the appropriate one but translating the EN to FR is tricky. Can you check please?

Reply:
Same question for the Ubika guide BTW

Reply:
Fixed with https://github.com/ovh/docs/pull/7064/commits/bf8431c197d37cda38568d8c537b1faaedff3e30
Thanks!